app/controllers/titles_controller.rb

```diff
@@ -15,6 +15,22 @@ def index
            meta: { totalResults: @titles.totalResults }
   end
 
+  def create
+    # RM API creates titles as a side-effect of creating resources, so
+    # we validate against a resource using the combined payload of the
+    # new title and an included resource with a packageId
+    resource_validation =
+      Validation::ResourceCreateParameters.new(combined_resource)
```

Comment:
so `ResourceCreateParameters` prior to this encompassed the title-level and resource-level attributes flattened into "our" concept of a resource?  why not split the validator in two now since we're splitting the concept (for updates, anyway)

Reply:
I think this is the way to go, but let's tackle that work, when we change titles to use the repository pattern.

Reply:
👍 for future reference between now and the date that is approaching, unless i explicitly mark something as "NO.  NO NO NO." you can safely assume i'm a) asking something out of curiosity or b) making notes so we don't forget to address things later on.  i'm totally fine with waiting.